industry_benchmarks/utils/extras/extract_results.py

```diff
@@ -40,12 +41,18 @@ def get_type(res):
 
 def load_results(f):
     # path to deserialized results
-    result = json.load(open(f, 'r'), cls=JSON_HANDLER.decoder)
+    with open(f, 'r') as fd:
+        result = json.load(fd, cls=JSON_HANDLER.decoder)
     if result['estimate'] is None or result['uncertainty'] is None:
-        errmsg = (
-            f"Calculations for {f} did not finish successfully!"
-            )
-        raise ValueError(errmsg)
+        # Keeping this check so if we do hit an error somehow, we print the traceback
+        click.echo(f"Calculations for {f} did not finish successfully!")
```

Comment:
It's a bit redundant to keep this around, but also at the same time it's a nice function to re-use elsewhere, so I'm happy to keep it there!

Reply:
I had the same thought